libkbfs/tlf_journal.go

```diff
@@ -1170,6 +1170,12 @@ func (j *tlfJournal) removeFlushedMDEntry(ctx context.Context,
 func (j *tlfJournal) flushOneMDOp(
 	ctx context.Context, end MetadataRevision,
 	maxMDRevToFlush MetadataRevision) (flushed bool, err error) {
+	if maxMDRevToFlush == MetadataRevisionUninitialized {
+		// Short-cut `getNextMDEntryToFlush`, which would otherwise read
+		// an MD from disk and sign it unnecessarily.
```

Comment:
We can probably add a method to the MD journal to get the earliest ordinal (== revision) and also use that too. Maybe add a TODO?

Also it's a bit weird to have both `maxMDRevToFlush` and `end`, which seem to me to do the same thing. Maybe add a TODO to fix that?